package.json

```diff
@@ -74,6 +74,7 @@
     "concurrently": "7.0.0",
     "cross-env": "7.0.3",
     "dotenv-flow": "3.2.0",
+    "eslint-plugin-unicorn": "43.0.2",
```

Comment:
We had this plugin mentioned in two workspaces, I moved it up. Eventually we want to keep it in`@hashintel/eslint-config` only, but this requires changes in half of the packages. ATM, only blocks extend their ESLint config from this package.